pkg/stream/pipeline.go

```diff
@@ -149,11 +151,17 @@ func (p *Pipeline) process(ctx context.Context, force bool) {
 	}()
 
 	p.ticker.Stop()
-	messages, err := p.callback.Process(ctx, p.batch)
 
-	if err != nil {
-		p.logger.Error(err, "could not process the batch")
-		return
+	var err error
```

Comment:
why do you define this here? does it make a difference if you just deleted this and make it := in line 158 then?